bundles/audience_targeting.rst

```diff
@@ -12,42 +12,25 @@ target group of the visitor.
 Installation
 ------------
 
-Enable the bundle in `app/AbstractKernel.php`:
+Enable the bundle in `config/bundles.php`:
 
 .. code-block:: php
 
     <?php
-    abstract class AbstractKernel extends SuluKernel
-    {
-        // ...
-
-        public function reigsterBundles()
-        {
-            $bundles = [
-                // ...
-                new Sulu\Bundle\AudienceTargetingBundle\SuluAudienceTargetingBundle(),
-            ];
-
-            return $bundles;
-        }
-    }
+    Sulu\Bundle\AudienceTargetingBundle\SuluAudienceTargetingBundle::class => ['all' => true],
 
 Add the routes for the administration interface to the routing configuration
-file (`app/config/admin/routing.yml`):
+file (`config/routes/sulu_admin.yaml`):
```

Comment:
if we want to provide a recipe for this bundle we should move this routes into an own config file e.g: `sulu_audience_targeting_admin.yaml`

Reply:
I don't know if I would do that now.

1. If we do that, then the entire `sulu_admin.yaml` file should be gone.
2. If we support Symfony Flex, then these steps would be gone completely anyway.

Reply:
correct we have all other routes in the sulu_admin.yaml as well